internal/service/encryptionatrestprivateendpoint/resource_test.go

```diff
@@ -360,25 +293,61 @@ func configAzureBasic(projectID string, azure *admin.AzureKeyVault, region strin
 	return config
 }
 
-func checkAzureBasic(projectID string, azure *admin.AzureKeyVault, region string, expectApproved bool) resource.TestCheckFunc {
+func checkBasic(projectID, cloudProvider, region string, expectApproved bool) resource.TestCheckFunc {
 	expectedStatus := retrystrategy.RetryStrategyPendingAcceptanceState
 	if expectApproved {
 		expectedStatus = retrystrategy.RetryStrategyActiveState
 	}
+	attrsSet := []string{"id"}
+	if cloudProvider == "AZURE" {
+		attrsSet = []string{"id", "private_endpoint_connection_name"}
+	}
```

Comment:
```suggestion
	if cloudProvider == "AZURE" {
		attrsSet = append(attrsSet, "private_endpoint_connection_name")
	}
```
nit: feel this makes it easier to uderstand what computed attribute is common for all providers.